src/services/authStorage.ts

```diff
@@ -0,0 +1,113 @@
+import { LOCALSTORAGE_PREFIX } from 'global/const';
+
+const LOCALSTORAGE_AUTH_STORAGE_KEY = `${LOCALSTORAGE_PREFIX}auth`;
+const LOCALSTORAGE_USER_TOKENS_KEY = `${LOCALSTORAGE_PREFIX}userTokens`;
+
+const AUTH_STORAGE_CURRENT_ACCOUNT = 'account';
+const AUTH_STORAGE_AUTH_KEY = 'token';
+const AUTH_STORAGE_AUTH_REFRESH_KEY = 'refreshToken';
+
+class AuthStorage {
+  private static setAuth(value: string): void {
+    localStorage.setItem(LOCALSTORAGE_AUTH_STORAGE_KEY, JSON.stringify(value));
+  }
+
+  private static getAuth(): any {
+    return JSON.parse(localStorage.getItem(LOCALSTORAGE_AUTH_STORAGE_KEY) || '{}');
+  }
+
+  setAuthKey(value: string): void {
+    const auth = AuthStorage.getAuth();
+    auth[AUTH_STORAGE_AUTH_KEY] = value;
+    AuthStorage.setAuth(auth);
+  }
+
+  setAuthRefreshKey(value: string): void {
+    const auth = AuthStorage.getAuth();
+    auth[AUTH_STORAGE_AUTH_REFRESH_KEY] = value;
+    AuthStorage.setAuth(auth);
+  }
+
+  setCurrentAccount(value: string): void {
+    const auth = AuthStorage.getAuth();
+    auth[AUTH_STORAGE_CURRENT_ACCOUNT] = value;
+    AuthStorage.setAuth(auth);
+  }
+
+  // Updates user token list and token/refreshToken
+  updateAllTokens(authKey: string, authRefreshKey: string, currentAccount?: string) {
```

Comment:
Rename function, also everywhere you're calling it `authKey` and `authRefreshKey` it should just be `token` and `refreshToken`
```suggestion
  updateUserTokens(token: string, refreshToken: string, account?: string) {
```